mii/client.py

```diff
@@ -156,6 +160,38 @@ def destroy_session(self, session_id):
             client.destroy_session(session_id)
 
 
+class MIINonPersistentClient():
+    def __init__(self, task, deployment_name):
+        self.task = task
+        self.deployment_name = deployment_name
+
+    def query(self, request_dict, **query_kwargs):
+        task_methods = GRPC_METHOD_TABLE[self.task]
+        inference_pipeline = mii.non_persistent_models[self.deployment_name][0]
```

Comment:
We should throw an error that explains the reason when the `deployment_name` is not in the dictionary.

The another possible design is to return a client object from `deploy()` (or a new function like `deploy_non_persistent()`. The user wouldn't need to be aware of the deployment name, which is globally and internally stored in MII.

Reply:
I added an assertion for this in the query_handle() function.

Reply:
We still need to add an assertion here that `self.deployment_name in miii.non_persistent_models` because a user could still call `terminate` and then try run a query after, resulting in a potentially confusing error.

Reply:
Added it now, and removed the extra parameter in the query_handle().